src/backend/api/Fusion.Resources.Api/Notifications/IUrlResolver.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Api.Notifications
+{
+    public interface IUrlResolver
+    {
+        /// <summary>
+        /// Resolves url to active requests page
+        /// </summary>
+        /// <param name="orgProjectId">Pro org project id</param>
+        /// <param name="orgContractId">Pro org contract id</param>
+        /// <returns>The url or null if resolving fails</returns>
+        Task<string?> ResolveActiveRequests(Guid orgProjectId, Guid orgContractId);
+
+        /// <summary>
+        /// Resolves url to manage personnel page
+        /// </summary>
+        /// <param name="orgProjectId">Pro org project id</param>
+        /// <param name="orgContractId">Pro org contract id</param>
+        /// <returns>The url or null if resolving fails</returns>
+        Task<string?> ResolveManagePersonnel(Guid orgProjectId, Guid orgContractId);
```

Comment:
Not used as of now, since we decided that opening this page is not applicable in the case of a person being onboarded.
But it is available for use in case someone needs to add a link to this page in a future notification.